### DISCUSSION TITLE
Show only connections based on specific cable type, e.g. only power cables

### DISCUSSION THREAD

**Discussion by buzzingbren** [mattieserver/netbox-topology-views#395] [Q&A] [answered]
Hi all,

I love this Netbox Plugin, and I'm searching for a way to generate a topology which only show Power Cable connections.
This to help us visualize how all the power connections are divided over our PDUs and our Power Feeds. This will help us a lot, since we now have 4 separate power feeds and we would like to equalize the utilizations over these 4 feeds.
Since I'm unable to find a way to filter out the network connections, the view is filled with all kinds of connections (power and network alike).
Has anyone found a creative way to create a view with only the connections between devices and their PDUs?

**Comment by dreng** [comment, marked as answer]
Netbox Topology Views doesn't know about different cable types, but it knows about termination types. By default, NTV uses all possible types of termination like interfaces, front ports, console ports, power ports, etc.

If you like to ignore some of these termination types, you are able to set this as a preference.

- Navigate to Menu >> Topology Views >> Individual Options
- Use "Ignore Termination Types" in order to ignore everything but power ports and power outlets
- Click save button
- Go back to the topology, set filters (optional) and click the search button

**Reply by buzzingbren** [reply]
Thank you!

This was exactly what I needed! I was so fixated on cables, connections, cable types, etc... That this option of ignoring termination types flew right past me.

This worked perfect for me. Thanks!
